deliravision/torch/metrics.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import logging
+
+from deliravision.torch.utils.tensor_ops import make_onehot_npy
+
+
+logger = logging.getLogger(__file__)
+
+
+def compute_stat_score(pred, label, n):
```

Comment:
If this works on numpy, we should maybe have this in `vision_base`?

Reply:
Yes, you are right 👍